kolibri/core/discovery/api.py

```diff
@@ -16,6 +16,16 @@ class NetworkLocationViewSet(viewsets.ModelViewSet):
     serializer_class = NetworkLocationSerializer
     queryset = NetworkLocation.objects.all()
 
+    def filter_queryset(self, queryset):
```

Comment:
Could also use a FilterSet here with just `subset_of_users_device` as the only field.

Reply:
oh, I should have thought of that, much simpler, of course. Done